src/Lean/Server/FileWorker/WidgetRequests.lean

```diff
@@ -255,7 +255,7 @@ private def advanceTaggedTextHighlightState (text : String) (highlighted : α) :
 where
   updateState (text : String) (isHighlighted : Bool) : StateM TaggedTextHighlightState Unit :=
     modify fun s =>
-      let p : String.Pos.Raw := s.p + ⟨text.utf8ByteSize⟩
+      let p : String.Pos.Raw := s.p + text
```

Comment:
This one feels a bit weird to me

Reply:
Is the goal here not to advance `s.p` by the size of `text`?

Reply:
Sure, but this reads like it is adding a string to a position.

Reply:
Yes, this is the `HAdd String.Pos.Raw String String.Pos.Raw` which has existed for a long time and which is what you are "supposed" to use in this situation.